client/src/components/createSubmissionComponents/SubmissionUpload.js

```diff
@@ -31,10 +35,20 @@ const SubmissionUpload = () => {
   const onSubmit = async (e) => {
     e.preventDefault();
 
+    const uploadCount = [];
+
     const formData = new FormData();
     for (let i = 0; i < files.length; i++) {
       formData.append(`submissionPic`, files[i]);
     }
+
+    for (let pic of formData.entries()) {
+      uploadCount.push(pic);
+    }
+
+    if (uploadCount.length < 1) {
```

Comment:
Can you just do formData.entries().length and do away with uploadCount? If not, uploadCount should probably be an integer rather than storing a copy of all of the images

Reply:
Yes, I managed to prove that I can.